app/models/decidim/initiative.rb

```diff
@@ -0,0 +1,576 @@
+# frozen_string_literal: true
```

Comment:
Maybe this file can be extended rather than overriden ?

Reply:
For this one I encountered an unexpected issue, as I have to updates the `enum states:` it consider Active Support duplicates in my extend

By simplicity I kept the override, i tried to load with different way but always the same issue